tests/README.md

```diff
@@ -49,11 +49,11 @@ TC_PATHS="tests/kafkatest/tests/streams/streams_upgrade_test.py::StreamsUpgradeT
 ```
 * Run tests with a specific image name
 ```
-image_name="ducker-ak-openjdk:17-buster" bash tests/docker/run_tests.sh
```

Comment:
for another, perhaps we could use `openjdk:17` for this example to avoid using the explicit os version.